### PR TITLE
Use 200 status code for healthcheck endpoint 

### LINKED ISSUE #1282: Use 200 status code for healthcheck endpoint
**Is your feature request related to a problem? Please describe.**
My issue is related to https://github.com/meilisearch/meilisearch-kubernetes/issues/56. If you want to deploy this to GKE using there ingress you need a health check which returns a 200 status code

**Describe the solution you'd like**
Either a new health check endpoint or the existing one to return a 200 status code. I understand why 204 is used so I guess it could return 200 with the words success or similar.

**Describe alternatives you've considered**
The solution in the other ticket uses / as the endpoint for the health check but seems to not return 200 in production when using a master key.
The other solution I can think of is using Nginx to proxy the 204 and change it to a 200 code but this is another dependency which should not be required.

### PR DESCRIPTION
**Pull request by irevoire**
closes  #1282

### CODE REVIEW

**Review by curquiza — CHANGES_REQUESTED** [review]
Thanks for your PR!

Could you please replace "status: ok" by "status: available"? We chose to follow the IBM convention -> https://github.com/meilisearch/MeiliSearch/issues/1282#issuecomment-799424875

**Review by curquiza — APPROVED** [review]

File: meilisearch-http/src/routes/health.rs [hunk 1]
```diff
@@ -9,5 +9,6 @@ pub fn services(cfg: &mut web::ServiceConfig) {
 
 #[get("/health")]
 async fn get_health() -> Result<HttpResponse, ResponseError> {
-    Ok(HttpResponse::NoContent().finish())
+    let payload = serde_json::json!({ "status": "available" });
+    Ok(HttpResponse::Ok().body(payload.to_string()))
```
Comment (MarinPostma) [inline meilisearch-http/src/routes/health.rs:13]: Can you change to use the json method please?

**Review by MarinPostma — CHANGES_REQUESTED** [review]

**Review by MarinPostma — APPROVED** [review]
Lgtm

**Comment by MarinPostma** [comment]
bors merge

**Pull request merged**